docs/integrations/destinations/redshift.md

```diff
@@ -51,18 +51,17 @@ For COPY strategy:
     to objects in the staging bucket.
 - **Secret Access Key**
   - Corresponding key to the above key id.
-- **S3 Filename pattern**
```

Comment:
We should expand on what the filename pattern does and perhaps highlight here that this may be 'required' in some use cases for the COPY strategy. Also see lines under 'Optional parameters': where we might want to expand this discussion.

Reply:
I think that this is something we can investigate and enhance later. Added to my issue tracking doc.